invenio_deposit/views/rest.py

```diff
@@ -220,6 +222,7 @@ def __init__(self, serializers, pid_type, ctx, *args, **kwargs):
         for key, value in ctx.items():
             setattr(self, key, value)
 
+    @require_api_auth()
```

Comment:
and modify the function that implement the `read_permission_factory` to do the job?